packages/tools/src/utilities/viewport/scrollToSlice.ts

```diff
@@ -0,0 +1,60 @@
+import { Types, getEnabledElement, StackViewport } from '@cornerstonejs/core';
+
+type Metadata = {
+  imageIdIndex?: number;
+  /**
+   * The position of the camera in world space
+   */
+  cameraPosition?: Types.Point3;
+  /**
+   * The focal point of the camera in world space
+   */
+  cameraFocalPoint?: Types.Point3;
+  /**
+   * The normal on which the tool was drawn
+   */
+  viewPlaneNormal?: Types.Point3;
+  /**
+   * The viewUp on which the tool was drawn.
+   */
+  viewUp?: Types.Point3;
+  /**
+   * The FrameOfReferenceUID
+   */
+  FrameOfReferenceUID: string;
+};
+
+/**
+ * It uses the metadata to scroll to the slice that is intended. Currently,
+ * only supports imageIdIndex in the metadata, but could be extended to
+ * support other types of metadata such as camera position, focal point, etc.
+ * for volume viewports.
+ * @param element - the HTML Div element scrolling inside
+ * @param metadata - the metadata used for scrolling
+ * @returns Promise that resolves to ImageIdIndex
+ */
+function scrollToSlice(
+  element: HTMLDivElement,
+  metadata: Metadata
```

Comment:
Metadata is hugely overloaded term, can we call this something like options, with a type of `ScrollToSliceOptions`/`JumpToSliceOptions` or something?